pandas/core/reshape/merge.py

```diff
@@ -1705,7 +1706,11 @@ def flip(xs):
             left_values = left_values.view("i8")
             right_values = right_values.view("i8")
             if tolerance is not None:
-                tolerance = tolerance.value
+                try:
```

Comment:
this is not a good way
rather just wrap it in Timedelta then leave the original